doc/DEMO.md

```diff
@@ -264,37 +312,10 @@ echo $PWD/stake_1_data
 ```
 and paste the result as the data dir.
 - Update the `coordinator_host` with `127.0.0.1:8383`, `coordinator_noise_key` with the key obtained in step 2
-- Update the `stakeholders_xpubs`: it will contain all the stakeholder xpubs created in the ceremony
-
-```
-stakeholders_xpubs = [
-    *first stakeholder xpub*,
-    *second stakeholder xpub*,
-]
-```
-Please make sure that the keys are in the right order, the first as first and the second as second. Not doing so could lead to tricky bugs.
-- Update the `managers_xpubs`: it will contain all the manager xpubs created in the ceremony:
-
-```
-managers_xpubs = [
-    *first manager xpub*,
-]
-```
-
-- Update the `cosigners_keys`: it will contain all the cosigner keys created in step 3
-
-```
-cosigners_keys = [
-    *first cosigner key*,
-    *second cosigner key*,
-]
-```
-Please make sure that the keys are in the right order, the first as first and the second as second. Not doing so could lead to tricky bugs.
-
+- Update the `scripts_config`: replace the `deposit_descriptor`, `unvault_descriptor` and
```

Comment:
Update it for manager 1 and stakeholder 2 too, please

Reply:
Well, no ? They use the same which they copied already?